collector/pkg/component/analyzer/cpuanalyzer/cpu_analyzer.go

```diff
@@ -116,10 +136,19 @@ func (ca *CpuAnalyzer) ConsumeTransactionIdEvent(event *model.KindlingEvent) {
 }
 
 func (ca *CpuAnalyzer) analyzerJavaTraceTime(ev *TransactionIdEvent) {
+	javatracekey := &JavaTracesKey{
+		TraceId: ev.TraceId,
+		PidString: ev.PidString,
+		StartTime: time.Now(),
```

Comment:
`time.Now()` cannot be considered as a part of a "key", because it changes all the time.